src/osm_configurator/view/toplevelframes/settings_frame.py

```diff
@@ -36,20 +35,32 @@ def __init__(self, state_manager, settings_controller):
             state_manager (state_manager.StateManager): The StateManager the frame will call, if it wants to switch states.
             settings_controller (settings_controller.SettingsController): Respective controller
         """
-        window = super().__init__()
+        super().__init__(
+            master=None,
+            width=frame_constants_i.FrameConstants.HEAD_FRAME_WIDTH.value,
+            height=frame_constants_i.FrameConstants.HEAD_FRAME_HEIGHT.value,
+            corner_radius=frame_constants_i.FrameConstants.FRAME_CORNER_RADIUS.value,
+            fg_color=frame_constants_i.FrameConstants.HEAD_FRAME_FG_COLOR.value
+        )
+
+        # Defining the grid
+        self.grid_columnconfigure(0, weight=1)
+        self.grid_columnconfigure(1, weight=1)
+        self.grid_rowconfigure(0, weight=1)
+        self.grid_rowconfigure(0, weight=1)
 
         self._state_manager = state_manager
         self._settings_controller = settings_controller
 
         self.title = "Settings Frame"
 
         self.settings_application_frame = \
-            SettingsApplicationFrame(self, self._settings_controller) \
-            .pack(side="top", padx=10, pady=10)
+            settings_application_frame_i.SettingsApplicationFrame(self, self._settings_controller)
+        self.settings_application_frame.grid(row=0, column=0, padx=10, pady=10)
 
         self.create_project_frame = \
-            SettingsProjectFrame(self, self._settings_controller) \
-            .pack(side="bottom", padx=10, pady=10)
+            settings_project_frame_i.SettingsProjectFrame(self, self._settings_controller)
+        self.create_project_frame.grid(row=1, column=0, padx=10, pady=10)
 
     def activate(self) -> bool:
```

Comment:
this method isn't quite correct, as not always both subFrames will be loaded, you need to ask if there is a project loaded currently or not, to find out if the project settings shall be visible or not